Reneiski_Homework1/Reneiski_Homework1/Lecture10/Man.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Reneiski_Homework1.Lecture10
+{
+    public class Man : Human
+    {
+        public string ManName { get; set; }
+        public Man(string firstname, string lastname) : base(firstname, lastname)
+        {
+            ManName = firstname;
+        }
+
+        public Man() : base("", "")
```

Comment:
just create parametless constructor in Human and you won't need this base("", "").
the same for Woman class